dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantMemoryStore.cs

```diff
@@ -376,6 +398,12 @@ public async Task RemoveWithPointIdBatchAsync(string collectionName, IEnumerable
         return (record.Item1, record.Item2);
     }
 
+    /// <inheritdoc/>
```

Comment:
This isn't part of the `IMemoryStore` interface.

Reply:
This is about the #1057